src/beautifiers/eslint.coffee

```diff
@@ -14,35 +14,35 @@ module.exports = class ESLintFixer extends Beautifier
   }
 
   beautify: (text, language, options) ->
-    context = this
-    return new @Promise((resolve, reject) ->
+    return new @Promise((resolve, reject) =>
       editor = atom.workspace.getActiveTextEditor()
       filePath = editor.getPath()
       projectPath = atom.project.relativizePath(filePath)[0]
 
       result = null
-      allowUnsafeNewFunction ->
+      allowUnsafeNewFunction =>
         importPath = Path.join(projectPath, 'node_modules', 'eslint')
+        eslint = null
         try
-          context.which('eslint').then((globalEslintPath) ->
-            if globalEslintPath != 'eslint'
-              importPath = Path.resolve(globalEslintPath, '../../lib/node_modules/eslint')
-            else
+          eslint = require(importPath)
+        catch
+          try
+            eslint = requireg('eslint')
```

Comment:
`requireg('eslint')` won't work here since it just calls `require('eslint')` first and only checks for a global eslint if none is provided with atom-beautify

Reply:
Thanks, I have installed the `requireg` package to atom-beautify.
Plz see my newest commit.

Reply:
How is installing requireg going to change is priority?

Reply:
I have change the priority in the newest update pr.
Plz check again.

Reply:
> requireg('eslint') won't work here since it just calls require('eslint') first and only checks for a global eslint if none is provided with atom-beautify

Instead of `requireg` looks like `import-global` should work: https://github.com/sindresorhus/import-global